### PR TITLE
chore: Update react-aria dependencies

### PR DESCRIPTION
**Pull request by ryan953**

### CODE REVIEW

File: static/app/components/comboBox/index.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import styled from '@emotion/styled';
 import {useComboBox} from '@react-aria/combobox';
 import {Item, Section} from '@react-stately/collections';
 import {type ComboBoxStateOptions, useComboBoxState} from '@react-stately/combobox';
+import type {Key} from '@react-types/shared';
```
Comment (evanpurkhiser) [inline static/app/components/comboBox/index.tsx:8]: What the heck are react-types

Comment (ryan953) [reply static/app/components/comboBox/index.tsx:8]: It's something from react-aria

**Comment by priscilawebdev** [comment]
PR looks good, but don't we need the 'package.json' file?

**Comment by ryan953** [comment]
This is getting stale. will open a new one

**Pull request closed without merging**